components/celplace/src/main/java/com/celements/crm/place/PlaceScriptService.java

```diff
@@ -10,10 +13,33 @@
 @Component("crmplace")
 public class PlaceScriptService implements ScriptService {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(PlaceScriptService.class);
+
   @Requirement
   private IGeocodingServiceRole geocodingService;
 
+  public ClassReference getGeoTagClassRef() {
+    try {
+      return geocodingService.getGeotagClass().getClassReference();
+    } catch (Exception exc) {
+      LOGGER.info("Failed to get getGeoTagClassRef.", exc);
+      return null;
+    }
+  }
+
+  /**
+   * @deprecated since 5.5 instead use getGeoTagClassRef
+   */
+  @Deprecated
   public ClassDefinition getGeotagClassDefinition() {
```

Comment:
This would be a breaking change. One release at least we need to keep it as deprecated. For more details see comment on https://synjira.atlassian.net/browse/CELDEV-896?focusedCommentId=38584

Reply:
this is a potential security issue since it exposes internal instances without any access protection.
please actively replace all usages with this PR and remove it.